docs/learn/encyclopedia/contract-development/types/built-in-types.mdx

```diff
@@ -24,9 +24,10 @@ Built-in types are available to all contracts for use as contract function input
 
 :::tip
 
-Custom types like structs, enums, and unions are also supported. See [Custom Types]. [Custom Types]: custom-types.mdx
+Custom types like structs, enums, and unions are also supported. See [Custom Types]. 
 
 :::
+[Custom Types]: custom-types.mdx
```

Comment:
I believe your build is failing because you need a blank line above this. It's a pain, I know. The `:::tip` kind of admonitions need some specific blank line padding:

- a blank line on either side of the opening `:::tip` tag
- a blank line on either side of the closing `:::` tag

```markdown
Some paragraph could be here. Then, I'll put a blank line:

:::tip

Blank lines around this part, too!

:::

This paragraph might talk about how there are so many blank lines to love!!
```

Reply:
Oh, this isn't really related to your change, huh? Don't know why the markdown formatting thing decided to do it now... 🤷🏻‍♂️